flowmachine/tests/test_cache_utils.py

```diff
@@ -0,0 +1,350 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+Tests for cache management utilities.
+"""
+from cachey import Scorer
+from unittest.mock import Mock
+
+import pytest
+
+from flowmachine.core import Table
+from flowmachine.core.cache import (
+    compute_time,
+    shrink_below_size,
+    shrink_one,
+    size_of_cache,
+    size_of_table,
+    score,
+    get_query_by_id,
+    get_cached_queries_by_score,
+    touch_cache,
+    get_max_size_of_cache,
+    set_max_size_of_cache,
+    get_cache_half_life,
+    set_cache_half_life,
+    invalidate_cache_by_id,
+)
+from flowmachine.features import daily_location
+
+
+def test_scoring(flowmachine_connect):
+    """Test that score updating algorithm is correct by comparing to cachey as reference implementation"""
+    dl = daily_location("2016-01-01").store().result()
+    dl_time = compute_time(flowmachine_connect, dl.md5)
+    dl_size = size_of_table(flowmachine_connect, *dl.table_name.split(".")[::-1])
+    initial_score = score(flowmachine_connect, dl.md5)
+    cachey_scorer = Scorer(1000)
+    cache_score = cachey_scorer.touch("dl", dl_time / dl_size)
+    assert cache_score == pytest.approx(initial_score)
+    # Touch again
```

Comment:
Add blank lines before comments to improve readability?
```suggestion
    # Touch again

```